### PR TITLE
Reloads dynamic window rules

### PR DESCRIPTION
**Pull request by MightyPlaza**
Reloads dynamic window rules on config reload

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -1564,6 +1564,14 @@ void CConfigManager::loadConfigLoadVars() {
     // Update window border colors
     g_pCompositor->updateAllWindowsAnimatedDecorationValues();
 
+    // Updates dynamic window rules
+    for (auto& w : g_pCompositor->m_vWindows) {
+        if(!w->m_bIsMapped)
+            continue;
+        
+        w.get()->updateDynamicRules();
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:1572]: wtf? `w->updateDynamicRules()`

File: src/config/ConfigManager.cpp [hunk 2]
```diff
@@ -1564,6 +1564,14 @@ void CConfigManager::loadConfigLoadVars() {
     // Update window border colors
     g_pCompositor->updateAllWindowsAnimatedDecorationValues();
 
+    // Updates dynamic window rules
+    for (auto& w : g_pCompositor->m_vWindows) {
+        if(!w->m_bIsMapped)
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:1569]: style: format the file with clang-format

**Comment by MightyPlaza** [comment]
fixed

**Review by vaxerski — APPROVED** [review]
wee woo

**Pull request merged**